src/stories/components/Line/Line.test.tsx

```diff
@@ -379,4 +380,20 @@ describe('Line', () => {
 			expect(point.getAttribute('stroke-width')).toEqual('2');
 		});
 	});
+
+	describe('Line animation tests', () => {
+		test('Line animates between congruent datasets', () => {
```

Comment:
Is this test running async or blocking?  We can probably use async to free up the test runner